CovertInfiltration/Src/CovertInfiltration/Classes/UIUtilities_Infiltration.uc

```diff
@@ -419,6 +419,30 @@ static function string GetAcademyTargetRank (optional XComGameState_Unit UnitSta
 	return class'X2ExperienceConfig'.static.GetRankName(class'X2Helper_Infiltration'.static.GetAcademyTrainingTargetRank(), '');
 }
 
+static function bool ShouldPauseGeoscape (int MilestoneReached)
```

Comment:
* The name is rather confusing, no context as to what it assumes
* Since it's only ever called from `XComGameState_MissionSiteInfiltration`, it belongs in that class, not in a shared one